app/models/form/sales/questions/deposit_and_mortgage_value_check.rb

```diff
@@ -0,0 +1,23 @@
+class Form::Sales::Questions::DepositAndMortgageValueCheck < ::Form::Question
+  def initialize(id, hsh, page)
+    super
+    @id = "deposit_and_mortgage_value_check"
+    @check_answer_label = "Deposit and mortgage against discount confirmation" # is this meant to come from somewhere or do I just write something sensible?
```

Comment:
Is this displayed anywhere? Can you omit it entirely?

Reply:
whoops I meant to put this comment in to remind me to investigate this properly. I don't know yet what check_answer_label is or why I was just copying the format from other files.

Reply:
I see what it is. Yes it can be displayed
<img width="931" alt="image" src="https://user-images.githubusercontent.com/51094020/214067160-03acef51-201c-40e3-a0f9-4df8644db19b.png">
I will assume that this label is acceptable on the basis that a label has not been provided in the ticket. But it seems there are quite a few places where devs are being allowed to make up copy